src/screens/Episode/components/Player/helpers/getPlayerTime.ts

```diff
@@ -0,0 +1,26 @@
+import * as dateFns from 'date-fns'
+import { DateFormatType, getFormattedDate } from 'src/helpers'
+import { UNIX_TIME_START, TIME_SEPARATOR } from '../../../common'
+
+const checkHourExist = (hours: string): boolean => {
```

Comment:
Actually this function contains only one line and we can avoid it. It will be better if we just create constant `HOUR_IS_NOT_EXISTS = "00"`